src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java

```diff
@@ -29,18 +37,55 @@ public class ElevatorConstants { // TODO: check real values
     public static final LoggedTunableNumber KP = new LoggedTunableNumber("kp");
     public static final LoggedTunableNumber KI = new LoggedTunableNumber("ki");
     public static final LoggedTunableNumber KD = new LoggedTunableNumber("kd");
+    public static final LoggedTunableNumber KV = new LoggedTunableNumber("kv");
+    public static final LoggedTunableNumber KA = new LoggedTunableNumber("ka");
+    public static final LoggedTunableNumber KG = new LoggedTunableNumber("kg");
 
     public static void initConstants() {
         switch (Constants.CURRENT_MODE) {
             case REAL:
                 KP.initDefault(0.8);
                 KI.initDefault(0.0);
                 KD.initDefault(0.0);
+                KV.initDefault(0.0);
+                KA.initDefault(0.0);
+                KG.initDefault(0.0);
             case SIM:
             case REPLAY:
                 KP.initDefault(43.0);
                 KI.initDefault(0.0003);
                 KD.initDefault(0.05);
         }
+
+        MAIN_MOTOR_CONFIGURATION
+                .withHardwareLimitSwitch(
+                        new HardwareLimitSwitchConfigs().withReverseLimitAutosetPositionValue(0))
+                .withMotionMagic(new MotionMagicConfigs())
```

Comment:
What's the point of motion magic configs if you don't change them?

Reply:
Also, I'm pretty sure motion magic expo has special configs.

Reply:
e4b8b8e0248b1edaccb683e868720136f25352fa